pnpm-workspace.yaml

```diff
@@ -47,30 +47,30 @@ catalogs:
         "@phosphor-icons/core": ^2.0.2
         "@popperjs/core": ^2.10.2
         "@khanacademy/mathjax-renderer": ^3.0.0
-        "@khanacademy/wonder-blocks-accordion": ^3.1.34
-        "@khanacademy/wonder-blocks-banner": ^4.2.14
-        "@khanacademy/wonder-blocks-button": ^11.0.0
-        "@khanacademy/wonder-blocks-clickable": ^7.1.20
-        "@khanacademy/wonder-blocks-core": ^12.3.0
-        "@khanacademy/wonder-blocks-data": ^14.1.4
-        "@khanacademy/wonder-blocks-dropdown": ^10.3.2
-        "@khanacademy/wonder-blocks-form": ^7.3.2
-        "@khanacademy/wonder-blocks-icon-button": ^10.3.15
-        "@khanacademy/wonder-blocks-icon": ^5.2.14
-        "@khanacademy/wonder-blocks-labeled-field": ^4.0.0
-        "@khanacademy/wonder-blocks-layout": ^3.1.31
-        "@khanacademy/wonder-blocks-link": ^9.1.20
-        "@khanacademy/wonder-blocks-pill": ^3.1.35
-        "@khanacademy/wonder-blocks-popover": ^6.1.24
-        "@khanacademy/wonder-blocks-progress-spinner": ^3.1.31
-        "@khanacademy/wonder-blocks-search-field": ^5.1.38
-        "@khanacademy/wonder-blocks-switch": ^3.3.13
+        "@khanacademy/wonder-blocks-accordion": ^3.1.37
+        "@khanacademy/wonder-blocks-banner": ^4.3.2
+        "@khanacademy/wonder-blocks-button": ^11.0.3
+        "@khanacademy/wonder-blocks-clickable": ^7.1.23
+        "@khanacademy/wonder-blocks-core": ^12.4.0
+        "@khanacademy/wonder-blocks-data": ^14.1.5
+        "@khanacademy/wonder-blocks-dropdown": ^10.3.7
+        "@khanacademy/wonder-blocks-form": ^7.3.5
+        "@khanacademy/wonder-blocks-icon-button": ^10.4.0
+        "@khanacademy/wonder-blocks-icon": ^5.2.17
+        "@khanacademy/wonder-blocks-labeled-field": ^4.0.3
+        "@khanacademy/wonder-blocks-layout": ^3.1.34
+        "@khanacademy/wonder-blocks-link": ^9.2.1
+        "@khanacademy/wonder-blocks-pill": ^3.1.38
+        "@khanacademy/wonder-blocks-popover": ^6.1.28
+        "@khanacademy/wonder-blocks-progress-spinner": ^3.1.34
+        "@khanacademy/wonder-blocks-search-field": ^5.1.42
+        "@khanacademy/wonder-blocks-switch": ^3.3.16
         "@khanacademy/wonder-blocks-theming": ^4.0.0
         "@khanacademy/wonder-blocks-timing": ^7.0.2
-        "@khanacademy/wonder-blocks-tokens": ^12.0.2
-        "@khanacademy/wonder-blocks-toolbar": ^5.1.33
-        "@khanacademy/wonder-blocks-tooltip": ^4.1.38
-        "@khanacademy/wonder-blocks-typography": ^4.2.16
+        "@khanacademy/wonder-blocks-tokens": ^12.2.0
```

Comment:
Updated WB dependencies by running `utils/sync\-dependencies\.ts ../frontend/pnpm-workspace.yaml`. This was so we can use the new WB tokens in perseus!